### PR TITLE
chore: Add shell to run l2node

### PR DESCRIPTION
**Pull request by Mdaiki0730**
## Description

closes: #XXXX

Add a shell to start L2 fullnode.

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: demo/run_l2node.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+make build && cp -r build/simd $GOPATH/bin/$L2NODE_BINARYNAME
+${L2NODE_BINARYNAME} version
+
+# Init sequencer
+
+${L2NODE_BINARYNAME} init rollupdemo --home $L2_KEYRING_DIR/$SEQUENCER_DIR --chain-id $L2_CHAIN_ID > /dev/null 2>&1
+${L2NODE_BINARYNAME} keys add validator --keyring-backend=test --home $L2_KEYRING_DIR/$SEQUENCER_DIR --recover --account=0 <<< ${TEST_MNEMONIC} > /dev/null 2>&1
+${L2NODE_BINARYNAME} add-genesis-account $(${L2NODE_BINARYNAME} --home $L2_KEYRING_DIR/$SEQUENCER_DIR keys show validator -a --keyring-backend=test) 100000000000stake,100000000000tcony --home $L2_KEYRING_DIR/$SEQUENCER_DIR > /dev/null 2>&1
+${L2NODE_BINARYNAME} gentx validator 10000000000stake --keyring-backend=test --home $L2_KEYRING_DIR/$SEQUENCER_DIR --chain-id=$L2_CHAIN_ID > /dev/null 2>&1
+${L2NODE_BINARYNAME} collect-gentxs --home $L2_KEYRING_DIR/$SEQUENCER_DIR > /dev/null 2>&1
```
Comment (tkxkd0159) [inline demo/run_l2node.sh:34]: Why don't you use $L2NODE_BINARYNAME instead of ${$L2NODE_BINARYNAME} for consistency?

And I don't recommend to throw out the result of command execution to track processing. If you thing it's too messy, you should refine them. Apart from that, why don't you show the results of the command execution as it is and shut it down in case of failure? (e.g. COMMAND || exit -1)

File: demo/run_l2node.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Run L2 sequencer
+${L2NODE_BINARYNAME} start --log_level "debug" --home $L2_KEYRING_DIR/$SEQUENCER_DIR --p2p.laddr "tcp://0.0.0.0:26555" --p2p.seeds="tcp://${TEST_SEQUENCER_P2P_ID}@127.0.0.1:26556" --rpc.laddr=tcp://0.0.0.0:26654 --grpc.address "0.0.0.0:9192" --grpc-web.address "0.0.0.0:9193" --rollkit.da_layer finschia --rollkit.da_config='{"rpc_uri":"'$RPC_URI'","chain_id":"'$L1_CHAIN_ID'","keyring_dir":"'$L1_KEYRING_DIR'","from":"'$TEST_SEQUENCER_ADDRESS'", "rollup_name":"'$ROLLUP_NAME'"}' --rollkit.namespace_id $NAMESPACE_ID  --rollkit.da_start_height $DA_BLOCK_HEIGH > $L2_KEYRING_DIR/$L2_CHAIN_ID.log 2>&1 &
```
Comment (tkxkd0159) [inline demo/run_l2node.sh:37]: Same

**Comment by ulbqb** [comment]
Can you construct test_sequenser.sh using run_l2node.sh?

**Review by ulbqb — APPROVED** [review]

**Review by tkxkd0159 — APPROVED** [review]
